src/conversations/ConversationList.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { StyleSheet, ListView } from 'react-native';
+
+import { getFullUrl } from '../utils/url';
+import { normalizeRecipients } from '../utils/message';
+import { isPrivateNarrow, isGroupNarrow } from '../utils/narrow';
+import UserItem from '../users/UserItem';
+import UserGroup from './UserGroup';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  groupHeader: {
+    fontWeight: 'bold',
+    paddingLeft: 8,
+    fontSize: 18,
+  }
+});
+
+export default class ConversationList extends Component {
+
+  props: {
+    conversations: string[],
+    realm: string,
+    users: any[],
+    onNarrow: (email: string) => void,
+  }
+
+  renderUserItem = ({ email, users, realm, narrow, onNarrow }) => {
```

Comment:
`renderUserItem` and `renderUserGroup` seem better suited as separate (stateless) components